source/gui/speechDict.py

```diff
@@ -0,0 +1,283 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2006-2021 NV Access Limited, Michael Curran, Peter Vágner, Mesar Hameed, Joseph Lee,
+# Aaron Cannon, Reef Turner, Ethan Holliger, Julien Cochuyt, Thomas Stivers, Cyrille Bougot
```

Comment:
This needs to be updated, referencing the discussions here: https://github.com/nvaccess/nvda/pull/12386